tests/installation/bootloader_zkvm.pm

```diff
@@ -85,7 +87,12 @@ sub run {
     record_info('VM instance', get_var('VIRSH_INSTANCE'));
     record_info('Guest ip', get_var('VIRSH_GUEST'));
 
-    return if (get_var('AGAMA'));
+    if (is_agama) {
```

Comment:
Why need to check the Agama installer start or not? Also for the next steps for 'login' and select_console 'installation'? For current test, we just return here, no more action needed for agama test.

Reply:
Good question, I keep the same logic as we do for sle15spx, and make sure live iso image is booted up and installer is started as well.

At the same time, It will not impact  yam job as well https://openqa.suse.de/tests/16291826#step/patch_agama_tests/1 [you need to access the live root as well here]

For `login` and `select_console`, you are correct, let me fix them.